packages/constants/src/spacing.js

```diff
@@ -77,3 +77,12 @@ export const SPACING_POINTS = {
   8: 50,
   9: 60,
 };
+
+export const WIDTHS = {
+  'one-quarter': '25%',
+  'one-third': '33.3333%',
+  'one-half': '50%',
```

Comment:
`one-` seems redundant and only seems to make code more verbose than necessary, especially in `one-half`. Do these names come from somewhere? Also 🚲

Reply:
they come from govuk-frontend
https://github.com/alphagov/govuk-frontend/blob/68bd09bb3e54b7ef4b4084ad2b3336858923a041/src/settings/_measurements.scss#L23

Having said that tho - those measurements settings are what's used for grid, rather than width.  You're right - they're redundant - and can maybe be got rid of, or supplemented with versions that don't include "one"

🚲 indeed :wink:

Reply:
Will add in reference to measurements link into the code